commands/bot_e2e_test.go

```diff
@@ -284,7 +284,7 @@ func (s *MmctlE2ETestSuite) TestBotDisableCmd() {
 		s.Require().Nil(appErr)
 
 		err := botDisableCmdF(s.th.Client, &cobra.Command{}, []string{newBot.UserId})
-		s.Require().Nil(err)
+		s.Require().NotNil(err)
```

Comment:
Please check for errors using `s.Require().Error(err)`, it makes the messages more readable.